contrib/ci/oss-fuzz.py

```
@@ -187,7 +187,7 @@ def compile(self, src: str, argv_extra: Optional[list[str]] = None) -> str:
             sys.exit(1)
         return os.path.join(self.builddir, f"{dst}")
 
-    def rustgen(self, src: str, includes: list[str] = []) -> str:
+    def rustgen(self, src: str, includes: List[str] = []) -> str:
```

Ruff (0.15.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)